buildspec.yml

```
@@ -19,28 +19,28 @@ phases:
     finally:
       - echo "Built artifacts:"
       - ls -lah ./bin/local
-      - ./bin/local/ecs-preview-amd64 --version
+      - ./bin/local/copilot-amd64 --version
   post_build:
     commands:
-      - VERSION=`./bin/local/ecs-preview-amd64 --version`
+      - VERSION=`./bin/local/copilot-amd64 --version`
       - VERSION=`echo $VERSION | grep -oE "[^ ]+$"`
       - COMMIT_VERSION=`git rev-parse --short HEAD`
       - echo "Creating latest and version-tagged artifacts..."
```

I realize this predates the current PR but do we actually create a "latest" artifact in this buildspec? It looks like we only generate the version-tagged ones.
```suggestion
      - echo "Creating version-tagged artifacts..."
```

---

Done!